services/supervisor/src/lib.rs

```diff
@@ -157,9 +161,9 @@ use exonum_rust_runtime::{
 
 use crate::{configure::ConfigureMut, mode::Mode};
 
+pub mod api;
```

Comment:
Nit: IMO, it makes sense not to re-export `api`-related types (i.e., `DeployInfoQuery` and `MigrationInfoQuery`) from the crate root.